cellx/networks/classifier.py

```diff
@@ -1 +1,13 @@
-# this will be the classifier
+# Building the CNN classifier:
+
+import tensorflow as tf
+from tensorflow import keras as K
+
+from cellx import layers
```

Comment:
```suggestion
from ..layers import Encoder2D
```
We can make this a relative import.

Reply:
@quantumjot I was just wondering whether you generally want us to go for relative imports when writing local import statements, since in this case an absolute statement like `from cellx.layers import Encoder2D` isn't too verbose and seems more readable according to PEP8?